cmd/ebrelayer/events/event.go

```diff
@@ -17,23 +17,34 @@ import (
 
 // LockEvent : struct which represents a single smart contract event
 type LockEvent struct {
-	Id    [32]byte
-	From  common.Address
-	To    []byte
-	Token common.Address
-	Value *big.Int
-	Nonce *big.Int
+	ChainID               *big.Int
```

Comment:
the ChainID should be a string

Reply:
This is an ethereum-specific ChainID, not a cosmos chain id, based on https://github.com/ethereum/EIPs/blob/master/EIPS/eip-155.md - it is used as an integer in the hashing and seems to be stored as an integer in most ethereum contexts (eg: https://github.com/ethereum-lists/chains/tree/master/_data/chains)

@denali Maybe should rename it to EthereumChainID?

Reply:
oh I see, yeah `EthereumChainID` is more clear

Reply:
Agreed, the distinction is important. Updated in https://github.com/cosmos/peggy/pull/68/commits/6accc479fa1a67730a93d884b608e7cd41816dd2